src/openforms/payments/contrib/worldline/migrations/0007_auto_20250829_1446.py

```diff
@@ -0,0 +1,101 @@
+# Generated by Django 4.2.23 on 2025-08-29 12:46
+
+from django.db import migrations
+from django.db.backends.base.schema import BaseDatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+
+import structlog
+
+from openforms.payments.contrib.ogone.constants import OgoneEndpoints
+from openforms.payments.contrib.worldline.models import WorldlineEndpoints
+
+logger = structlog.stdlib.get_logger(__name__)
+
+
+ENDPOINT_MAPPING = {
+    OgoneEndpoints.live: WorldlineEndpoints.live,
+    OgoneEndpoints.test: WorldlineEndpoints.test,
+}
+
+
+def migrate_merchants(apps: StateApps, schema_editor: BaseDatabaseSchemaEditor) -> None:
+    OgoneMerchant = apps.get_model("payments_ogone", "OgoneMerchant")
+    WorldlineMerchant = apps.get_model("payments_worldline", "WorldlineMerchant")
+    known_pspids = WorldlineMerchant.objects.values_list("pspid", flat=True)
+    ogone_merchants = OgoneMerchant.objects.all()
+    new_merchants = {}
+
+    for merchant in ogone_merchants:
+        if merchant.pspid in known_pspids:
+            logger.warning("skipped_merchant_migration", merchant=merchant)
```

Comment:
```suggestion
            logger.warning("skipped_merchant_migration", merchant=merchant.pspid)
```
avoid logging rich objects with structlog, as they will eventually be logged using `repr(...)` I believe. You should only pass JSON-serializable primitives as extra kwargs.